src/ReactiveSocket.cpp

```diff
@@ -75,6 +76,20 @@ void ReactiveSocket::requestSubscription(
   automaton->start();
 }
 
+void ReactiveSocket::requestFireAndForget(
```

Comment:
You won't know whether it was written anywhere (and why would it be helpful?), as everything on the way, including DuplexConnection, TCP, everything else may be "async". What is the information that this callback delivers? What can I do with that fact it was called?